hs_core/templates/403.html

```diff
@@ -0,0 +1,32 @@
+{% extends 'pages/page.html' %}
```

Comment:
I am not sure about the content of this page. @dtarb may want to look at this html code or want to test it if gets deployed for testing.